docs/serving/samples/knative-routing-go/routing.yaml

```diff
@@ -54,4 +54,6 @@ spec:
       # eventually be directed to LOgin service.
       - destination:
           host: istio-ingressgateway.istio-system.svc.cluster.local
+          port:
```

Comment:
could you also add the `port` to line https://github.com/knative/docs/pull/1671/files#diff-f0c3699af8b098b3338421148e699d4bL42 for `/search` ?

Reply:
Missed that one! Fixed.